### PR TITLE
Api improved list parameters

### PR DESCRIPTION
**Pull request by mmadariaga**
Some minor enhancements:
- allow to order by id
- use `like '%term%'` strategy on text search
- Expose API headers on CORS requests

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

File: library/Ivoz/Api/Swagger/Metadata/Resource/ResourceMetadata/FilterMetadataFactory.php [hunk 1]
```diff
@@ -74,16 +74,18 @@ private function getEntityFilters(string $resourceClass)
 
         $attributes = $this->getEntityAttributes($resourceClass);
         foreach ($attributes as $attribute) {
+            $type = $this->getFieldType($resourceClass, $attribute);
+            if (!is_null($type)) {
+                $filters['ivoz.api.filter.order'][$attribute] = Filter\OrderFilter::NULLS_LARGEST;
```
Comment (Kaian) [inline library/Ivoz/Api/Swagger/Metadata/Resource/ResourceMetadata/FilterMetadataFactory.php]: Have you considered creating constants for 'ivoz.api....' strings?

Comment (mmadariaga) [reply library/Ivoz/Api/Swagger/Metadata/Resource/ResourceMetadata/FilterMetadataFactory.php]: Done :)

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**